File: app/src/main/cpp/AudioPartInfo.cpp
//
// Created by cjf on 2017/5/2.
//

#include "AudioPartInfo.h"

AudioPartInfo::AudioPartInfo(long audioTime, std::string &filePath) {
    this->audioTime = audioTime;
    this->filePath = filePath;
}

long AudioPartInfo::getAudioTime() {
    return audioTime;
}

std::string& AudioPartInfo::getFilePath() {
    return filePath;
}

AudioPartInfo::~AudioPartInfo() {

}

File: app/src/main/java/lib/zxtcode/com/mylib/AudioPartInfo.java
package lib.zxtcode.com.mylib;

/**
 * Created by zxt lan4627@Gmail.com on 2018/1/24.
 */

public class AudioPartInfo {

    private String filePath;
    private long audioTime;

    public AudioPartInfo(String filePath, long audioTime) {
        this.filePath = filePath;
        this.audioTime = audioTime;
    }

    public String getFilePath() {
        return filePath;
    }

    public void setFilePath(String filePath) {
        this.filePath = filePath;
    }

    public long getAudioTime() {
        return audioTime;
    }

    public void setAudioTime(long audioTime) {
        this.audioTime = audioTime;
    }
}
